src/renderer/src/pages/lab/index.tsx

```
@@ -0,0 +1,5 @@
+const Lab = () => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)